pkg/api/dashboard_test.go

```diff
@@ -1009,7 +1009,8 @@ func TestDashboardAPIEndpoint(t *testing.T) {
 				return nil
 			})
 			bus.AddHandler("test", func(query *models.GetDashboardQuery) error {
-				query.Result = &models.Dashboard{Id: 1, Data: &simplejson.Json{}}
+				dataValue, _ := simplejson.NewJson([]byte(`{"id": 1, "editable": true, "style": "dark"}`))
```

Comment:
We should make sure the JSON parsing went OK:
```suggestion
				dataValue, err := simplejson.NewJson([]byte(`{"id": 1, "editable": true, "style": "dark"}`))
				require.NoError(t, err)
```

Reply:
Also, can we add a test for the error case?